_translator-files/po/fr/Installation-for-Linux.po

```diff
@@ -64,16 +65,15 @@ msgstr "{% include infobox_each_os.html %}"
 
 #. type: Plain text
 #: ../wiki/en/Installation-for-Linux.md:17
-#, fuzzy, no-wrap
-#| msgid "Upgrading? You may want to [back up your configuration](Software-Manual#backing-up-jamulus) first."
+#, no-wrap
 msgid "**Upgrading?** You may want to [back up your configuration](Software-Manual#backing-up-jamulus) first.\n"
-msgstr "Mise à jour ? Vous pouvez d'abord [sauvegarder votre configuration](Software-Manual#sauvegarde-de-jamulus)."
+msgstr "**Mise à jour ?** Vous pouvez d'abord [sauvegarder votre configuration](Software-Manual#sauvegarde-de-jamulus).\n"
```

Comment:
Needing `\n` is not nice...

Reply:
The translation is following the original (english) message. the '\n' should be removed from the source code first.